consoles/console.pm

```diff
@@ -74,10 +74,15 @@ sub screen {
 }
 
 # helper function
+# Keep ssh session for the maximum of ServerAliveCountMax x ServerAliveInterval seconds 
```

Comment:
delete whitespace at end of line, see https://travis-ci.org/github/os-autoinst/os-autoinst/builds/724766444#L842

Reply:
Done